src/app/shared/stache/stache.module.ts

```diff
@@ -34,7 +35,8 @@ import { StachePageSummaryModule } from './modules/page-summary/page-summary.mod
     StachePageSummaryModule,
     StachePageHeaderModule,
     StacheLayoutModule,
```

Comment:
I know you didn't do this, but these imports and exports should all be in alphabetical order.